### ISSUE TITLE
aws_dynamodb_table shows price warnings

### ISSUE THREAD

**Issue by alikhajeh1** [infracost/infracost#1194]
Both the latest release and the master branch seem to have an issue.

infracost v0.9.14:

```
Log: Multiple products found for aws_dynamodb_table.my_table Data storage, using the first product

 ├─ Data storage                                                                  Monthly cost depends on usage: $0.25 per GB
```

master branch: 

```
Log: No products found for aws_dynamodb_table.my_table Data storage, using 0.00`

 ├─ Data storage                                                                  Monthly cost depends on usage: $0.00 per GB
```

**Comment by vdmgolub** [comment]
@alikhajeh1 Could you provide the TF file example, please? I've tried to run an example in `us-east-1` and it didn't have any issues, integration tests don't fail either.

**Comment by aliscott** [comment]
This is now released in in v0.9.15.
